index.js

```diff
@@ -3,13 +3,17 @@
 	Author Tobias Koppers @sokra
 */
 var path = require("path");
+var loaderUtils = require("loader-utils");
+
 module.exports = function() {};
 module.exports.pitch = function(remainingRequest) {
+  var options = Object.assign({}, loaderUtils.getOptions(this));
```

Comment:
No need `Object.assign` here, rewrite this to `getOptions(this) || {}`

Reply:
It was my initial suggestion to do this. Thinking about it better, is a good idea to keep `loaderUtils.getOptions(this) || {};` as if the loader get called many times, it will have the performance decreased. I
l'll left a new comment suggestion to rever this @aymasse